lib/home_dash_web/components/cards/default.ex

```diff
@@ -1,11 +1,9 @@
 defmodule HomeDashWeb.Cards.Default do
   use HomeDashWeb, :html
 
-  import HomeDashWeb.CardsCommon
-
   attr :card, HomeDash.Card, required: true
   attr :title, :string, required: false
-  attr :class, :string, required: false
+  attr :class, :string, default: nil, required: false
```

Comment:
I do not think we need `default` and `required`. Those are somewhat conflicting messages.